packages/yew-macro/src/html_tree/html_block.rs

```diff
@@ -7,11 +7,11 @@ use syn::parse::{Parse, ParseStream};
 use syn::{braced, token};
 
 pub struct HtmlBlock {
-    content: BlockContent,
+    pub(crate) content: BlockContent,
```

Comment:
Is there a reason you went for `pub(crate)` instead of just `pub`? Macro crates can't currently export any members anyway so I usually just stick to `pub`.

Reply:
Not really.